.github/workflows/codex-plugin-scanner.yml

```
@@ -0,0 +1,35 @@
+jobs:
+  scan:
+    name: Plugin Scanner
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4.2.2
+
+      - name: Run scanner
+        uses: hashgraph-online/hol-codex-plugin-scanner-action@b45d6b583afe05819b24edc8e6418c9ad2e1f1d0 # v1
+        with:
```

actionlint (1.7.12)

[error] "with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)